classes/class-wc-connect-service-settings-store.php

```diff
@@ -42,6 +42,14 @@ public function get_shared_settings() {
 			);
 		}
 
+		public function get_origin_address() {
+			return get_option( 'woocommerce_connect_origin_address', array() );
```

Comment:
We are using the pattern `wc_connect_` for our other options. Let's do that here too